knit.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+#
+
+knit.pattern=example-[a-zA-Z0-9-]+-##\\.kt
```

Comment:
We have `knit` properties in a root folder, in `docs` and in `knit` project.
Could we please reduce the count of configuration files for Knit?

Reply:
I dropped a separate knit module and directory. It is way leaner now. All the core knit configs are in `docs` folder now (where they are actually used).